app/Context/Deposit/Application/Command/GetTransactionExplorerLink/GetTransactionExplorerLinkHandler.ts

```diff
@@ -0,0 +1,26 @@
+import GetTransactionExplorerLink from "./GetTransactionExplorerLink";
+import ExternalBlockchainRepositoryInterface from "../../../Domain/ExternalBlockchain/RepositoryInterface";
+import etherscanLink from "@metamask/etherscan-link";
+import Web3Repository from "../../../Infrastructure/ExternalBlockchain/Web3Repository";
+export default class GetTransactionExplorerLinkHandler {
```

Comment:
Providing data is not a job for the Command. A Query should be used here.

Event better, it can be a method of a model Domain/ExternalBlockchain/Contract.

In the client code we will use `session.externalContract?.transactionExplorerLink()`.